mountinfo/mountinfo_linux.go

```diff
@@ -173,7 +173,7 @@ func unescape(path string) (string, error) {
 	}
 
 	// The following code is UTF-8 transparent as it only looks for some
-	// specific characters (backslach and 0..7) with values < utf8.RuneSelf,
+	// specific characters (backslash and 0..7) with values < utf8.RuneSelf,
```

Comment:
should we enable `misspell` in golang-ci-lint? https://golangci-lint.run/usage/linters/